### PR TITLE
Add filter to cli command 'proposals'

### PR DESCRIPTION
**Pull request by tcharding**
Currently when using the cli and running command `proposals` we get all
the proposals back.  It would be nice to be able to narrow down the
search i.e using a filter.  We can filter based on the country and also
the provider ID.

Add filter term to command `proposals`.  No documentation added.  Needs
extending to include something in the output of `help` to show that this
feature exists.

With this applied
```
» proposals SG
2018-10-23T05:59:50.896303068 [Info] [Mysterium.api] Proposals fetched: 41
[INFO] Found 41 proposals (filter: 'SG')
[INFO] - provider id: 0x44ceda7262fcc48d591ee468c8a7319585d51aee, proposal id: 1, country: SG
[INFO] - provider id: 0xeed4ee266b2175a7f4063ecd59eef414b60f9d1d, proposal id: 1, country: SG
[INFO] - provider id: 0xf03f364719d09b37260872735c113baaee4f4f1a, proposal id: 1, country: SG
```

Signed-off-by: Tobin C. Harding <me@tobin.cc>

### CODE REVIEW

**Comment by tcharding** [comment]
It would be nice to get this
```
» list
[INFO] Mysterium CLI tequilapi commands:
  connect 
  identities 
  ├── new 
  ├── list 
  status 
  healthcheck 
  proposals [filter-term]
...
```

File: cmd/commands/cli/command.go [hunk 1]
```diff
@@ -291,18 +291,29 @@ func (c *cliApp) healthcheck() {
...
 		if country == "" {
 			country = "Unknown"
 		}
+
 		msg := fmt.Sprintf("- provider id: %v, proposal id: %v, country: %v", proposal.ProviderID, proposal.ID, country)
-		info(msg)
+
+		if filter == "" ||
+			strings.Contains(proposal.ProviderID, filter) ||
+			strings.Contains(country, filter) {
```
Comment (soffokl) [inline cmd/commands/cli/command.go]: If it contains the only country, maybe `filter` is not a best name? Why not call it `country` or something similar?

Comment (zolia) [reply cmd/commands/cli/command.go]: this filters not just for the country, but ProviderID itself, we might leave it 'filter'.

Comment (soffokl) [reply cmd/commands/cli/command.go]: true, missed it somehow, sorry.

**Comment by tcharding** [comment]
Looks like this was a tooling issue again.  New version of `goimports` put the import list in a different order.  Let's see how travis likes it now.

File: cmd/commands/cli/command.go (hunk 1 above)
Comment (tcharding) [reply cmd/commands/cli/command.go]: Other functions that accept an argument use `argString` for the argument identifier - I used 'filter' here to be more descriptive.

**Comment by tcharding** [comment]
No love from Travis today.  Doesn't look like travis is running the build now and the very useful error message I can see is ` The Travis CI build could not complete du...`

Is there a limit on number of times a non-member  can push to a PR perhaps?

**Pull request merged**